go/pkg/yolosvc/driver_github.go

```diff
@@ -38,9 +39,10 @@ func (svc *service) GitHubWorker(ctx context.Context, opts GithubWorkerOpts) err
 	opts.applyDefaults()
 
 	worker := githubWorker{
-		svc:    svc,
-		opts:   opts,
-		logger: opts.Logger.Named("ghub"),
+		svc:         svc,
+		opts:        opts,
+		logger:      opts.Logger.Named("ghub"),
+		repoConfigs: parseRepoConfigs(opts.Repos),
 	}
 
```

Comment:
here you can do two opts checking:
1. validate if there is some inconsistency and raise a warning (logger) or an error (return)
2. `return nil` when there is not misconfiguration but when you know that it's useless to start a github worker (if repos list is empty -> no need to run the rest of the program

I suggest something like this that you can implement:

```suggestion
shouldRun, err := worker.ValidateConfig()
if err != nil {
return err
}
if !shouldRun {
return nil
}
```